data/anaconda.conf

```diff
@@ -64,6 +64,15 @@ ignored_packages =
 # Enable installation of latest updates.
 enable_updates = True
 
+# List of .treeinfo variant types to enable.
+# Valid items:
+#
+#   addon
+#   optional
+#   variant
+#
+enabled_repositories_from_treeinfo = addon optional variant
```

Comment:
The file `scientific-linux.conf` doesn't change the default settings. Is it correct?

Reply:
That is correct.